lib/matplotlib/colors.pyi

```diff
@@ -138,6 +138,96 @@ class ListedColormap(Colormap):
     def resampled(self, lutsize: int) -> ListedColormap: ...
     def reversed(self, name: str | None = ...) -> ListedColormap: ...
 
+class MultivarColormap:
+    name: str
+    colormaps: list[Colormap]
```

Comment:
I don't see this attribute.

Reply:
Good catch. Originally there was this interface, but I decided on letting __getitem__ be the only interface to the constituent colormaps. i.e. I removed the line 143.